scripts/manager_power_kw.lua

```diff
@@ -184,6 +183,73 @@ function getDomainPowerCastAction(nodeAction)
 	return rAction;
 end
 
+-- Get action data for the actions lists on the reference sheet for martial advantages
+-- It's basicalily a copy of getPCPowerAction without the requirement for an actor, and without handling 'cast' type rolls
+function getMartialAdvantagePowerAction(nodeAction, sSubRoll)
```

Comment:
I think this is fine as is, just maybe worth keeping an eye out as a potential location for user feedback if they have other extensions that they would expect to work for martial advantage powers.